core/src/main/java/hudson/tools/DownloadFromUrlInstaller.java

```diff
@@ -122,15 +122,16 @@ protected DescriptorImpl() {
         }
 
         protected Downloadable createDownloadable() {
-            return new Downloadable(getId());
+            return new Downloadable(getDownloadableId());
```

Comment:
This broke the tool installer for NodeJS Plugin: [JENKINS-28093](https://issues.jenkins-ci.org/browse/JENKINS-28093)